openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -355,6 +355,14 @@ Workspace ForwardTranslator::translateModelPrivate( model::Model & model, bool f
     globalGeometryRules.setString(openstudio::GlobalGeometryRulesFields::RectangularSurfaceCoordinateSystem, "Relative");
     m_idfObjects.push_back(globalGeometryRules);
 
+    {
+      auto zoneAirContaminantBalances = model.getConcreteModelObjects<ZoneAirContaminantBalance>();
```

Comment:
@kbenne I didn't understand why the following lines were not working:

https://github.com/NREL/OpenStudio/blob/2109-trans-contaminant-balance/openstudiocore/src/energyplus/ForwardTranslator.cpp#L2727

https://github.com/NREL/OpenStudio/blob/2109-trans-contaminant-balance/openstudiocore/src/energyplus/ForwardTranslator.cpp#L400

Did you test that this change works?  It seems like it would be the same to me.